app/services/ratings.js

```diff
@@ -0,0 +1,43 @@
+const { User, Rating } = require('../models');
+const { sequelize } = require('../models');
+const logger = require('../logger');
+
+exports.findRateWeet = async (id, ratingUserId) => {
+  const rating = await Rating.findOne({ where: { weetId: id, ratingUserId } });
+  return rating;
+};
+
+exports.createRateWeet = async rateWeetData => {
+  let transaction = {};
+  const { ratingUserId, weetId, score, userId, newPoints } = rateWeetData;
+  try {
+    transaction = await sequelize.transaction();
+    const rateWeet = await Rating.create({ ratingUserId, score, weetId }, { transaction });
+    await User.update({ points: newPoints }, { where: { id: userId }, individualHooks: true, transaction });
```

Comment:
What would happen if 2 or more users rate a weet at the same time? I think some of the point will not be added to the weet owner user.